CONTRIBUTING.md

```diff
@@ -4,7 +4,7 @@ Pull requests for bug fixes are always welcome!
 
 Before submitting new features or changes to current functionality, it is recommended to first
 [open an issue](https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/new)
-and discuss your ideas or propose the changes you wish to make.
+and [discuss](https://github.com/open-telemetry/opentelemetry-java-instrumentation/discussions/new?category=q-a) your ideas or propose the changes you wish to make.
```

Comment:
To me the intent of `it is recommended to first open an issue and discuss your ideas` is that you open an issue and that issue is used for discussion. Linking the `discuss` to opening a discussion in that sentence does, in my opinion, not make sense as it leaves an impression that you should open both an issue and a discussion.